db/system_keyspace.cc

```diff
@@ -606,8 +606,15 @@ schema_ptr system_keyspace::built_indexes() {
             {"bytes_out", long_type},
```

Comment:
The commit message has to be updated, the offset thing is no longer required, system-schema versions are now hash based and are automatically updated when the table changes.

Reply:
Ah, true. Forgot to change it.